frontend/web/components/feature-override/FeatureOverrideCTA.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react'
+import Utils from 'common/utils/utils'
+import Button from 'components/base/forms/Button'
+import { removeUserOverride } from 'components/RemoveUserOverride'
+import Icon from 'components/Icon'
+import {
+  FeatureState,
+  IdentityFeatureState,
+  ProjectFlag,
+} from 'common/types/responses'
+import { useHasPermission } from 'common/providers/Permission'
+
+type FeatureOverrideCTAType = {
+  level: 'identity' | 'segment'
+  hasUserOverride: boolean
+  projectFlag: ProjectFlag
+  environmentId: string
+  identity?: string
+  identifier?: string
+  environmentFeatureState: FeatureState
+  overrideFeatureState?: FeatureState | IdentityFeatureState | null
+}
+
+const FeatureOverrideCTA: FC<FeatureOverrideCTAType> = ({
+  environmentId,
+  hasUserOverride,
+  identifier,
+  identity,
+  level,
+  overrideFeatureState,
+  projectFlag,
+}) => {
+  const { permission, permissionDescription } =
+    Utils.getOverridePermission(level)
+  const { permission: hasPermission } = useHasPermission({
+    id: environmentId,
+    level: 'environment',
+    permission,
+  })
+  switch (level) {
+    case 'identity': {
+      if (!hasUserOverride) {
+        return null
+      }
+      return (
+        hasUserOverride && (
+          <>
+            {Utils.renderWithPermission(
+              hasPermission,
+              permissionDescription,
```

Comment:
Isn't it missing `Constants.environmentPermissions(` ?

Reply:
Ah yep it is!